plugin/tower/pkg/controller/policy/controller.go

```diff
@@ -972,10 +972,12 @@ func (c *Controller) parseSecurityPolicy(securityPolicy *schema.SecurityPolicy)
 		}
 		// generate intra group policy
 		policy, err := c.generateIntragroupPolicy(securityPolicy.GetID(), policyMode, &securityPolicy.ApplyTo[item])
-		if err != nil || policy == nil {
+		if err != nil {
 			return nil, err
 		}
-		policyList = append(policyList, *policy)
+		if policy != nil {
```

Comment:
if policy is nil, only bypass is enough?

Reply:
if `policyList` doesn't have the policy, the reconciler will delete it in `kube-apiserver`.